CHANGELOG.md

```diff
@@ -1,19 +1,20 @@
 # Change Log
 
-## Unreleased
+## 0.6.0 - 2021-04-13
 ### Added
 - Now depends on the "Azure Resources" extension, which provides a "Resource Groups" and "Help and Feedback" view
+- IntelliSense for `staticwebapp.config.json` file
 
 ### Changed
 - "Report an Issue" button was removed from errors. Use the "Help and Feedback" view or command palette instead
 - Minimum version of VS Code is now 1.53.0
 - Icons updated to match VS Code's theme. Install new product icon themes [here](https://marketplace.visualstudio.com/search?term=tag%3Aproduct-icon-theme&target=VSCode)
-
-## 0.5.1 - 2021-04-12
-
-### Added
+- Build presets pre-generate values for app, api, and output locations that must be confirmed
 - Experimental framework to test incremental changes
```

Comment:
It looks like these fellas used to be in the "Added" section but now they're in "Changed". Was that intentional?